wgsl/index.bs

```diff
@@ -8071,11 +8071,18 @@ Atomically stores the value `v` in the atomic object pointed to
 `atomic_ptr` and returns the original value stored in the atomic object.
 
 ```rust
-atomicCompareExchangeWeak(atomic_ptr: ptr<SC, atomic<T>, A>, cmp: T, v: T) -> vec2<T>
+atomicCompareExchangeWeak(atomic_ptr: ptr<SC, atomic<T>, A>, cmp: T, v: T) -> _atomic_compare_exchange_result<T>
+
+struct _atomic_compare_exchange_result<T> {
+  old_value : T; // old value stored in the atomic
+  replaced : bool; // true if the exchange was done
```

Comment:
Maybe this should be called `exchanged` given the name of the intrinsic (and comment?)

Reply:
good point!